docs/DevGuide/CodeReviewGuide.md

```diff
@@ -53,6 +52,10 @@ Stylistic Items:
   `or`, `and`, and `not` instead of `||`, `&&`, and `!`.
 * Use C-style Doxygen comments (`/*! ... */`) when using multi-line math,
   otherwise C-style and C++ style comments are accepted.
+* Use the `align` environment instead of `equnarray`. See the
+  [texfaq](https://texfaq.org/FAQ-eqnarray) for an explanation as to why.
+* Multi-line equations must have a blank Doxygen line before and after the
+  equation.
```

Comment:
Perhaps add `...so clang-format won't attempt to re-format them.` also here (take or leave)

Reply:
We generally use the guidelines as a "Do this" without justification because otherwise it's too long and then even fewer people will read it. The justification for in the longer Dox tutorial so I think we should keep this short. :man_shrugging: